app/imports/api/users/methods.js

```diff
@@ -10,35 +10,52 @@ import { SimpleSchema } from 'meteor/aldeed:simple-schema';
 
 
 const sendNotificationEmail = function(user) {
+  // find the emails of any admin users
+  let base_admin_users = Meteor.users.aggregate([{$match: {roles: 'admin'}},
+                                                 {'$group':
+                                                     {_id: null, admin_emails:
+                                                         { $addToSet:
+                                                          "$profile.email"}}}])[0].admin_emails;
+
+  // get the email notification list
+  let notification_list;
   if(_.isEmpty(Meteor.settings.email) || _.isEmpty(Meteor.settings.email.notification_list)) {
     console.error("Notification list is not configured");
-    return;
+    //return;
+    notification_list = [];
+  } else {
+    notification_list = Meteor.settings.email.notification_list;
   }
-  let template = "A new user has registered for an account:\n" +
-    "\n" +
-    "Email: " + user.email + "\n" +
-    "Name: " + user.name + "\n" +
-    "Organization(s): " + user.organization.join(", ") + "\n" +
-    "\n" + 
-    "You can approve the account by logging in and visiting " + 
-    Meteor.absoluteUrl("users") +
-    "\n" +
-    "Thanks!\n" +
-    "IOOS Registry";
+  // combine the addresses of the two, ignoring any duplicates
+  const admin_and_notification_list = new Set([...base_admin_users,
```

Comment:
Getting errors here when registering/inserting a new user:
=> Meteor server restarted                    
I20180921-08:52:52.891(-4)? Exception while invoking method 'users.insert' TypeError: Can't call method on  undefined
I20180921-08:52:52.893(-4)?     at TypeError (native)
I20180921-08:52:52.893(-4)?     at module.exports (/home/user1/.meteor/packages/ecmascript-runtime/.0.3.15.1v5c3e++os+web.browser+web.cordova/npm/node_modules/core-js/modules/_defined.js:3:28)
I20180921-08:52:52.

Also had to add explain: true otherwise other errors:
  let base_admin_users = Meteor.users.aggregate([{$match: {roles: 'admin'}},
                                                 {'$group':
                                                     {_id: null, admin_emails:
                                                         { $addToSet:
                                                          "$profile.email"}}}], {explain: true})[0].admin_email